pkgs/by-name/bi/bitbox-bridge/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  libudev-zero,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bitbox-bridge";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "BitBoxSwiss";
+    repo = "bitbox-bridge";
+    tag = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-+pMXWXGHyyBx3N0kiro9NS0mPmSQzzBmp+pkoBLH7z0=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-6vD0XjGH1PXjiRjgnHWSZSixXOc2Yecui8U5FAGefBU=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    libudev-zero
+  ];
```

Package currently does not build on darwin because of this dependency

```suggestion
  buildInputs = stdenv.hostPlatform.isLinux [
    libudev-zero
  ];
```

---

Fixed, thanks for pointing it out!